linux/Instructions_for_desktop_app.md

```diff
@@ -0,0 +1,4 @@
+# Instuctions for Setting Up your Desktop File For BlueBubbles in Linux
+1. edit the `Bluebubbles.desktop` change exec to the location of the `bluebubbles_app`
```

Comment:
Minor nitpick but this should say something like "In `Bluebubbles.desktop`, change exec to..."